fiboa_cli/datasets/commons/ec.py

```diff
@@ -80,11 +81,11 @@ def get_code_column(self, gdf):
         return col if col.dtype == 'object' else col.astype(str)
 
     def add_hcat(self, gdf):
-        ec_mapping = load_ec_mapping(self.ec_mapping_csv, url=self.mapping_file)
+        self.ec_mapping = load_ec_mapping(self.ec_mapping_csv, url=self.mapping_file)
```

Comment:
Maybe I misunderstand the purpose of the change, but isn't this to only load the data once? In this case, there's a condition missing that only runs this line in case the variable is None....